src/pages/lesson-details/LessonsDetails.styles.ts

```diff
@@ -13,12 +12,25 @@ export const styles = {
     gap: '10px'
   },
   attachment: {
-    background: palette.basic.grey,
+    background: 'basic.grey',
     borderRadius: '5px',
     p: '16px 24px',
     display: 'flex',
     justifyContent: 'space-between'
   },
+  button: {
+    marginLeft: 'auto',
+    backgroundColor: 'basic.grey',
+    color: 'basic.black',
+    maxWidth: '97px',
```

Comment:
what if for example the translation language changes and the button text becomes longer, better set the width according to content, and use padding if needed

Reply:
done